core/src/test/scala/sttp/openai/requests/files/FilesResponseDataSpec.scala

```diff
@@ -34,4 +34,22 @@ class FilesResponseDataSpec extends AnyFlatSpec with Matchers with EitherValues
     // then
     givenResponse.value shouldBe expectedResponse
   }
+
+  "Given delete file response as Json" should "be properly deserialized to case class" in {
+    // given
+    val listFilesResponse = fixtures.FilesResponse.deleteFileJsonResponse
+    val expectedResponse = DeletedFileData(
```

Comment:
**question:** Shouldn't you use the fixture here?

Reply:
I guess I'm using it right here `val listFilesResponse = fixtures.FilesResponse.deleteFileJsonResponse`